### DISCUSSION TITLE
struggling with SMG-5410

### DISCUSSION THREAD

**Discussion by cm2962** [mbuchber/ha_heliotherm#8] [Q&A]
Hi Michael, hi all,
first of all, great project and l like to see progress on Heliotherm improvements :)

Unfortunately, I am not getting it up and running w/ the SMG-5410

Is there potentially anybody using this setup already and could guide me? I am using a 16 yr old heliotherm heatpump.

Many thanks
Christian

**Comment by cm2962** [comment]
just wanted to follow up on this.
Nobody here having the SMG-5410 up and running and could give me a hint?
Thanks

**Reply by Snited** [reply]
i also tried many days bit could not get it to work with SMG-5410. Now i am using normal htrest with a raspi connectet to the heatpump. not very comfortable but works

**Reply by cm2962** [reply]
Cool, I thought about it as alternative. Had it up and running (just in swagger) about a year ago.
Would you be so kind and share your HA configuration? I am not yet that firm with doing this on my own. Thanks Christian

**Reply by Snited** [reply]
I have this in my configuration.yaml (copy&paste will not work -> github changes format):

rest:
  - resource: http://192.168.177.108:8777/api/v1/device/
    sensor:
      - name: "Heliotherm Seriennummer"
        unique_id: Heliotherm_Seriennummer
        value_template: "{{ value_json.serial_number }}"
      - name: "Heliotherm Softwareversion"
        unique_id: Heliotherm_Softwareversion
        value_template: "{{ value_json.software_version }}"
  - resource: http://192.168.177.108:8777/api/v1/param/
    scan_interval: 60
    sensor:
      - name: Heliotherm Außentemperatur
        unique_id: Heliotherm_Außentemperatur
        value_template: "{{ value_json['Temp. Aussen'] }}"
        unit_of_measurement: °C
        device_class: temperature
      - name: Heliotherm Warmwasser Solltemperatur
        unique_id: Heliotherm_WW_SollTemp
        value_template: "{{ value_json['WW Normaltemp.'] }}"
        unit_of_measurement: °C
        device_class: temperature
      - name: Heliotherm Raum Solltemperatur
        unique_id: Heliotherm_Raum_SollTemp
        value_template: "{{ value_json['HKR Soll_Raum'] }}"
        unit_of_measurement: °C
        device_class: temperature
      - name: Heliotherm Temperatur Vorlauf
        unique_id: Heliotherm_Temp_Vorlauf
        value_template: "{{ value_json['Temp. Vorlauf'] }}" 
        unit_of_measurement: °C
        device_class: temperature
      - name: Heliotherm Temperatur Rücklauf
        unique_id: Heliotherm_Temp_Rücklauf
        value_template: "{{ value_json['Temp. Ruecklauf'] }}"
        unit_of_measurement: °C
        device_class: temperature
      - name: Heliotherm Temperatur Warmwasser
        unique_id: Heliotherm_Temp_WW
        value_template: "{{ value_json['Temp. Brauchwasser'] }}"
        unit_of_measurement: °C
        device_class: temperature
